projects/packages/my-jetpack/_inc/components/onboarding-screen/connection-form/connection-form.module.scss

```
@@ -0,0 +1,67 @@
+.connection-form {
+	display: flex;
+	flex-direction: column;
+	gap: 16px;
+	width: 100%;
+	max-width: 450px;
+}
+
+.title {
+	font-size: 32px;
+	font-weight: 500;
+}
+
+.description {
+	font-size: 16px;
+	font-weight: 400;
+	color: var(--jp-gray-100, #101517);
+}
+
+.social-button {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	align-self: stretch;
+	margin-bottom: 0;
+	padding: 14px 16px;
+	width: 100%;
+	border: 1px solid #cccccc;
```

```suggestion
	border: 1px solid #ccc;
```

Let's use a short version where possible.